lib/msw-config.ts

```diff
@@ -52,6 +44,8 @@ type MirageServer = {
     options?: unknown
   ) => (request: RestRequest) => ResponseData | PromiseLike<ResponseData>;
 
+  shouldLog: () => boolean;
```

Comment:
I'm only defining parts of the `MirageServer` we actually care about / use here, to allow this to be as decoupled from upstream Mirage as possible.